packages/app/client/src/ui/atoms/Image.js

```diff
@@ -5,20 +5,20 @@ import { makeStyles } from '@material-ui/core/styles'
 const useStyles = makeStyles(() =>
   ({
     image: props => ({
-      borderRadius: 5,
+      borderRadius: props.borderRadius,
       objectFit: props.objectFit,
       height: props.height,
       width: props.width
     })
   })
 )
 
-const Image = ({ height, width, objectFit, children, ...props}) => {
+const Image = ({ height, width, sameSize, objectFit, children, borderRadius = 5, ...props}) => {
```

Comment:
Could you could  use the Image.defaultProps  to use borderRadius default value, please ?